src/EventStore.Core/Services/Transport/Enumerators/Enumerator.AllSubscriptionFiltered.cs

```diff
@@ -374,6 +381,9 @@ public class AllSubscriptionFiltered : IAsyncEnumerator<ReadResponse> {
 									_subscriptionId, _eventFilter, caughtUp);
 
 								confirmationPositionTcs.TrySetResult(new TFPos(caughtUp, caughtUp));
+
+								_tracker.AddSubscription(_subscriptionId, null, caughtUp);
+
```

Comment:
ive not looked properly, but would it make sense to make these tracker calls in the subscriptionsservice so that the three kinds of subscription (stream, all, allfiltered) are handled together?